src/Analyzers/CSharp/Tests/MakeFieldReadonly/MakeFieldReadonlyTests.cs

```diff
@@ -1705,6 +1705,23 @@ public async Task TestVolatileField3()
 {
     private volatile object first;
     private readonly object second;
+}");
+        }
+
+        [WorkItem(33009, "https://github.com/dotnet/roslyn/issues/33009")]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/33009"), Trait(Traits.Feature, Traits.Features.CodeActionsMakeFieldReadonly)]
```

Comment:
📝 #33009 relates to ref _returns_, but this one is a new issue related to ref _locals_. Can you file a new issue and update these two links?

Reply:
➡️ #46785

Reply:
@sharwell Done.

Do you want me to update the comment for the suppression as well? (That file currently not included in the PR at all)